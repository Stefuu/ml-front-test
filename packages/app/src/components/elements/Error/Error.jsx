import React from 'react'
import Flex from '../../elements/Flex'

const Error = () => (
  <Flex
    justifyContent='center'
    alignItems='center'
    width='100vw'
    height={300}
    fontSize={26}>
    <span>Se produjo un error, vuelva a intentarlo m√°s tarde</span>
  </Flex>
)

export default Error
