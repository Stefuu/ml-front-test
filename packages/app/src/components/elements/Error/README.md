# Error

## Error message in case API fails
