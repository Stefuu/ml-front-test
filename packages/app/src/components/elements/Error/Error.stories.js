import React from 'react'
import { storiesOf } from '@storybook/react'
import { doc } from 'storybook-readme'
import readmeFile from './README.md'
import centered from '@storybook/addon-centered'
import { withKnobs } from '@storybook/addon-knobs'
import theme from '../../themes/default'
import { ThemeProvider } from 'styled-components'
import Error from './Error'

storiesOf('app/components/elements/Error', module)
  .addParameters({ jest: ['Button.test.js'] })
  .add('README', doc(readmeFile))
  .addDecorator(centered)
  .addDecorator(withKnobs)
  .addParameters({ jest: ['Content.test.js'] })
  .add('simple', () => (
    <ThemeProvider theme={theme}>
      <Error />
    </ThemeProvider>
  ))
