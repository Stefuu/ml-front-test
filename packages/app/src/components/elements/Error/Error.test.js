import React from 'react'
import { render } from 'react-testing-library'
import Error from './Error'

describe('Error', () => {
  it('Should render [snapshot]', async () => {
    const { container } = render(<Error />)
    expect(container.firstChild).toMatchSnapshot()
  })
})
