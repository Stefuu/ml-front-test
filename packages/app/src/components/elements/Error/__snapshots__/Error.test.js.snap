// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Error Should render [snapshot] 1`] = `
<div
  class="sc-bdVaJa cxfQT"
>
  <span>
    Se produjo un error, vuelva a intentarlo m√°s tarde
  </span>
</div>
`;
